### DISCUSSION TITLE
No matching certificate found for every requested profile

### DISCUSSION THREAD

**Discussion by Tr736** [codemagic-ci-cd/codemagic-docs#2394] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Using Yaml config instead of workflow

### Expected results

To build

### Actual results

No matching certificate found for every requested profile is thrown. 

When I change from workflow and use a codemagic.yaml file it fails with the above

### Build id (optional)

65df3749205e8627c5981ff9

**Comment by dtrdic** [comment]
hey @Tr736 can you please share the entire URL so I can take a closer look, as I don't see any logs on this build id

**Reply by Tr736** [reply, marked as answer]
Hey @dtrdic 

 all good. The error message wasn’t exactly helpful but it was because my provisioning profile didn’t have the right certificate. Happy to close this down

**Reply by dtrdic** [reply]
Awesome, nice to hear that!
﻿Alright, I'll close this thread since the issue has been resolved. If you encounter any other issues don't hesitate to contact us.
﻿Have a nice day!
